From 4fd69d6616b07a652089d71a74d11fb4a8da0049 Mon Sep 17 00:00:00 2001
From: Wang ShaoBo <bobo.shaobowang@huawei.com>
Date: Sat, 27 Feb 2021 18:35:31 +0800
Subject: [PATCH] openeuler_defconfig: Enable MPAM by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 4fd69d6616b07a652089d71a74d11fb4a8da0049


hulk inclusion
category: feature
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

Enable MPAM by default.

Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 1743f53a8bd0..02e5e8503e8a 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -7077,3 +7077,6 @@ CONFIG_CC_HAS_SANCOV_TRACE_PC=y
 # CONFIG_MEMTEST is not set
 # end of Kernel Testing and Coverage
 # end of Kernel hacking
+CONFIG_MPAM=y
+CONFIG_RESCTRL=y
+CONFIG_ACPI_MPAM=y
-- 
2.26.2

